internal/build/generate.go

```diff
@@ -46,22 +47,41 @@ func GenerateGo(zones map[string]*Zone) error {
 		z.TagBits = tagBits(tagValues, z.Tags)
 	}
 
-	data := struct {
-		Zones     map[string]*Zone
-		TLDs      map[string]*Zone
-		Domains   []string
-		Offsets   map[string]int
-		TagType   string
-		Tags      []string
-		TagValues map[string]uint64
-	}{
-		zones,
-		tlds,
-		domains,
-		offsets,
-		tagType,
-		tags,
-		tagValues,
+	data := templateData{
+		Zones:     zones,
+		TLDs:      tlds,
+		Domains:   domains,
+		Offsets:   offsets,
+		TagType:   tagType,
+		Tags:      tags,
+		TagValues: tagValues,
+		Strings:   []string{},
+	}
+
+	// Pre-index strings and string slices
+	for _, d := range domains {
+		data.domainString(d)
+	}
+	for _, d := range domains {
+		data.domainStringSlice(zones[d].NameServers)
+	}
+	for _, d := range domains {
+		data.domainStringSlice(zones[d].Wildcards)
+	}
+	for _, d := range domains {
+		data.domainString(zones[d].WhoisServer)
+	}
+	for _, d := range domains {
+		data.urlString(zones[d].WhoisURL)
```

Comment:
We'll need to get the forthcoming RDAP server URLs into this list as well.

Reply:
That’s the idea.

Reply:
Yup, will handle that separately once the RDAP support is merged. It's a trivial add.